plugins/modules/azure_rm_openshiftmanagedcluster_kubeconfig_info.py

```diff
@@ -0,0 +1,235 @@
+#!/usr/bin/python
+#
+# Copyright (c) 2020 Haiyuan Zhang <haiyzhan@micosoft.com>
+#
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+
+DOCUMENTATION = '''
+---
+module: azure_rm_openshiftmanagedcluster_kubeconfig_info
+version_added: '1.17.0'
+short_description: Get admin kubeconfig of Azure Red Hat OpenShift Managed Cluster
+description:
+    - get kubeconfig of Azure Red Hat OpenShift Managed Cluster instance.
+options:
+    resource_group:
+        description:
+            - The name of the resource group.
+        required: true
+        type: str
+    name:
+        description:
+            - Resource name.
+        required: true
+        type: str
+    path:
+        description:
+            - Destination filepath of kubeconfig file
+        required: false
+        type: str
+extends_documentation_fragment:
+    - azure.azcollection.azure
+    - azure.azcollection.azure_tags
```

Comment:
Arguments are not supported in the parameters, but tags are defined in the documentation, please delete!
```suggestion
```